website/content/docs/v0.7/Guides/configuring-the-cluster-endpoint.md

```diff
@@ -30,9 +31,9 @@ The configuration can either be done on a Loadbalancer, or simply trough DNS.
 For example:
 
 > This is in the config file for the cluster e.g. init.yaml, controlplane.yaml and join.yaml.
-> for more details, please see: [v1alpha1 endpoint configuration](https://www.talos.dev/docs/v0.7/en/configuration/v1alpha1#controlplane)
+> for more details, please see: [v1alpha1 endpoint configuration](https://www.talos.dev/docs/v0.6/en/configuration/v1alpha1#controlplane)
```

Comment:
is 0.6 version in 0.7 configs intended?
There are many other places where 0.6 is referenced from 0.7.

Reply:
whoops, good catch :+1:

Reply:
Fixed.